src/registers/autonegotiate_sink.rs

```diff
@@ -678,4 +679,19 @@ mod tests {
         let bytes = actual.as_bytes();
         assert_eq!(bytes, &AutonegotiateSink::DEFAULT);
     }
+
+    #[test]
+    fn test_from_pps_request_interval() {
+        // Test simple values
+        assert_eq!(PpsRequestInterval::EightSeconds, 0x0.into());
+        assert_eq!(PpsRequestInterval::FourSeconds, 0x1.into());
+        assert_eq!(PpsRequestInterval::TwoSeconds, 0x2.into());
+        assert_eq!(PpsRequestInterval::OneSecond, 0x3.into());
+
+        // Verify that only the low 2-bits are considered
+        assert_eq!(PpsRequestInterval::EightSeconds, 0xC.into());
+        assert_eq!(PpsRequestInterval::FourSeconds, 0xD.into());
+        assert_eq!(PpsRequestInterval::TwoSeconds, 0xE.into());
+        assert_eq!(PpsRequestInterval::OneSecond, 0xF.into());
```

Comment:
The entire `u8::MIN..=u8::MAX` is small enough that every value can be directly tested
```suggestion
        // verify values outside of 0..=3 are accepted and don't panic
        for value in u8::MIN..=u8::MAX {
            let _ = PpsRequestInterval::from(value); // should not panic
        }
```

Reply:
Done.